src/argilla/client/feedback/training/schemas/base.py

```diff
@@ -660,6 +699,14 @@ def __label2id__(self):
     def __id2label__(self):
         return self.label.question.__id2label__
 
+    @property
+    def label(self):
+        return self.defaults.label
+
+    @property
+    def text(self):
+        return self.defaults.text
```

Comment:
Maybe we could add some typing hints here

Reply:
check